R/stores.R

```diff
@@ -355,16 +361,33 @@ HttpStore <- R6::R6Class("HttpStore",
     options = NULL,
     headers = NULL,
     client = NULL,
+    zmetadata = NULL,
+    cache_time_seconds = 3600,
     make_request = function(item) {
-      # Remove leading slash if necessary.
-      if(substr(item, 1, 1) == "/") {
-        key <- substr(item, 2, length(item))
-      } else {
-        key <- item
-      }
+      key <- item_to_key(item)
+
+      # per-session http get cache
+      mem_get <- memoise::memoise(\(client, path) client$get(path), 
```

Comment:
Will this get re-memoized on each call to `make_request()`? Can we create `mem_get` in the constructor, and then store it as a variable on the instance?

Reply:
That's a good question -- probably. I'll make that change quick.